cloud/deployment/deployment_variable.go

```diff
@@ -59,7 +59,7 @@ func VariableList(deploymentID, variableKey, ws, envFile string, useEnvFile bool
 
 // this function modifies a deployment's environment variable object
 // it is used to create and update deployment's environment variables
-func VariableModify(deploymentID, variableKey, variableValue, ws, envFile string, useEnvFile, makeSecret, updateVars bool, client astro.Client, out io.Writer) error {
+func VariableModify(deploymentID, variableKey, variableValue, ws, envFile string, variableList []string, useEnvFile, makeSecret, updateVars bool, client astro.Client, out io.Writer) error {
```

Comment:
i know it's probobly unrelated to your change ... but we need reduce amount of arguments and refactor similar places via better abstraction and structs.
It's not a blocker to merge your PR, just general idea.

cc @neel-astro